images/nginx/Makefile

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 
 # 0.0.0 shouldn't clobber any released builds
-TAG ?= 0.64
+TAG ?= 0.65
 REGISTRY ?= quay.io/kubernetes-ingress-controller
 ARCH ?= $(shell go env GOARCH)
 DOCKER ?= docker
 
-ALL_ARCH = amd64 arm arm64 ppc64le s390x
+ALL_ARCH = amd64 arm arm64 ppc64le
```

Comment:
I saw https://github.com/openresty/lua-nginx-module/pull/1379 today - that tells me ingress-nginx isn't going to work on ARM64 because lua-nginx-module does not support it yet.

Reply:
there's also a user reported issue about this https://github.com/kubernetes/ingress-nginx/issues/2802